mysql/docker-entrypoint-initdb.d/02_data.sql

```diff
@@ -356,4 +365,88 @@ CREATE PROCEDURE INSERTROWS(NUM INT) BEGIN
 
 DELIMITER ;
 
-CALL InsertRows(250);
\ No newline at end of file
+CALL InsertRows(250);
+
+CREATE TABLE `Labels` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `label` varchar(50) CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci NOT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `Unique_label` (`label`) USING BTREE
+)
+;
+
+INSERT INTO playerdata.Labels (label) VALUES
+    ('Agility_bot'),
+    ('Barrows_bot'),
+    ('Blast_mine_bot'),
+    ('Clue_Scroll_bot'),
+    ('Construction_Magic_bot'),
+    ('Cooking_bot'),
+    ('Crafting_bot'),
+    ('Fishing_bot'),
+    ('Fishing_Cooking_bot'),
+    ('Fletching_bot'),
+    ('Herblore_bot'),
+    ('Hunter_bot'),
+    ('LMS_bot'),
+    ('Mage_Guild_Store_bot'),
+    ('Magic_bot'),
+    ('Mining_bot'),
+    ('mort_myre_fungus_bot'),
+    ('Phosani_bot'),
+    ('PVM_Melee_bot'),
+    ('PVM_Ranged_bot'),
+    ('PVM_Ranged_Magic_bot'),
+    ('Real_Player'),
+    ('Runecrafting_bot'),
+    ('Smithing_bot'),
+    ('Soul_Wars_bot'),
+    ('temp_real_player'),
+    ('test_label'),
+    ('Thieving_bot'),
+    ('Unknown'),
+    ('Unknown_bot'),
+    ('Vorkath_bot'),
+    ('Wintertodt_bot'),
+    ('Woodcutting_bot'),
+    ('Woodcutting_Firemaking_bot'),
+    ('Woodcutting_Mining_bot'),
+    ('Zalcano_bot'),
+    ('Zulrah_bot')
+;
+
+SELECT 
+    pl1.id AS voter_id, 
+    pl2.id AS subject_id,
+    pr.prediction,
+    pr.predicted_confidence,
+    "" AS feedback_text,
+    CASE WHEN RAND() > 0.5 THEN 1 ELSE -1 END AS vote,
+    (SELECT label FROM Labels ORDER BY RAND() LIMIT 1) AS proposed_label
+FROM (SELECT * FROM Players ORDER BY RAND() LIMIT 1000) pl1
+JOIN (SELECT * FROM Players ORDER BY RAND() LIMIT 1000) pl2 ON pl1.id <> pl2.id
+JOIN Predictions pr ON pr.id = pl2.id
+ORDER BY RAND()
+LIMIT 100
+;

```

Comment:
duplicate

Reply:
yep you are right

Reply:
resolved by @RusticPotatoes
[remove unneeded select](https://github.com/Bot-detector/public-api/pull/25/commits/de1cdba08613b58f5163d666e6f03b5f5d659933)